### DISCUSSION TITLE
What is Linux specific behavior ?

### DISCUSSION THREAD

**Discussion by haoadoreorange** [piroor/treestyletab#3017] [Q&A] [answered]
Sorry for the wrong place of the post. I can't find any info about "Linux specific behavior". What is it about ? What behaviors ?

**Comment by irvinm** [comment]
https://github.com/piroor/treestyletab/discussions might be a better place to ask this.

**Comment by piroor** [comment, marked as answer]
There are only some trivial behaviors for Linux:

* Some labels of localized messages around desktop notifications. They cannot handle click events on notifications themselves so TST says like "click the button to grant..." only on Linux.
* Disabled clearing of drop position marker when the mouse leaves the sidebar.
